nix/sources.json

```diff
@@ -1,15 +1,15 @@
 {
     "candid": {
-        "branch": "master",
+        "branch": "claudio/candid-subtype-check",
```

Comment:
is `candid-subtype-check` already merged to `master`?

Reply:
Nope, it's still waiting in PR https://github.com/dfinity/candid/pull/323

Reply:
Done